### PR TITLE
fix failures request in contract manager 

### PR DESCRIPTION
**Pull request by quasisamurai**
[TASK](https://hadronlabs.atlassian.net/browse/NTRN-359)

Related PRs
- [njs](https://github.com/neutron-org/neutronjs/pull/3)
- [integration tests](https://github.com/neutron-org/neutron-integration-tests/pull/335)

### CODE REVIEW

**Review by joldie777 — DISMISSED** [review]

File: proto/neutron/contractmanager/query.proto [hunk 1]
```diff
@@ -46,6 +46,13 @@ message QueryParamsResponse {
 
 // QueryFailuresRequest is request type for the Query/Failures RPC method.
 message QueryFailuresRequest {
+  // address of the contract which Sudo call failed.
+  string address = 1;
+  cosmos.base.query.v1beta1.PageRequest pagination = 3;
+}
+
+// QueryFailureRequest is request type for the Query/Failures RPC method.
+message QueryFailureRequest {
```
Comment (NeverHappened) [inline proto/neutron/contractmanager/query.proto:55]: `pagination` field seems to be not needed logically (and not used) for `QueryFailureRequest`.
Please remove it

File: proto/neutron/contractmanager/query.proto [hunk 2]
```diff
@@ -18,7 +18,7 @@ service Query {
   }
 
   // Queries a Failure by contract address and failure ID.
-  rpc AddressFailure(QueryFailuresRequest) returns (QueryFailuresResponse) {
+  rpc AddressFailure(QueryFailureRequest) returns (QueryFailuresResponse) {
```
Comment (NeverHappened) [inline proto/neutron/contractmanager/query.proto]: How about making `QueryFailureResponse` struct without pagination? It seems strange that we return it in array even tho it's always one failure. And pagination always null

**Review by NeverHappened — CHANGES_REQUESTED** [review]

**Comment by NeverHappened** [comment]
Also - please attach PR to neutronjs here

**Review by NeverHappened — APPROVED** [review]

**Comment by quasisamurai** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/10584317289

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**